### PR TITLE
Completed Exercise 6

### PR DESCRIPTION
**Pull request by TreshaSar**

### CODE REVIEW

File: sarabia-exercise6.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+def largerThan(list, n):
+    result = []
+    for item in list:
+      count += 1
+      if item > n:
+        if item not in result:
+          (result.append(list))
+    print("List of numbers " + str(list))
+    result.sort
+    print("Numbers that are greater than " + str(n))
```
Comment (mlhaus) [inline sarabia-exercise6.py:20]: If n is 60, your code will print `Numbers that are greater than 60`. You also need to print the result array list.

File: sarabia-exercise6.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+def main():
+
+  howMany = getNum("Please enter a number between 10 and 20.", 10.0, 20.0)
```
Comment (mlhaus) [inline sarabia-exercise6.py:25]: Decimals are not necessary here. This number will be used to determine how many random numbers to generate. For example, you can't generate 15.5 numbers.

**Review by mlhaus — CHANGES_REQUESTED** [review]
Looks good. Make these two corrections to get full credit.

**Comment by TreshaSar** [comment]
**I fixed the 2 areas and removed the decimals to get full credit for this assignment. Thank you for the feedback!**  👍

**Pull request merged**